geopyspark/geotrellis/catalog.py

```diff
@@ -16,199 +17,34 @@
-                                                   parameter_dict['keyspace'],
-                                                   parameter_dict['table'])
-
-        elif backend == 'hbase':
-
-            # The assumed uri looks like: hbase://zoo1, zoo2, ..., zooN: port/table
-            (zookeepers, port) = parsed_uri.netloc.split(':')
-            table = parsed_uri.path
-
-            if 'master' in options:
-                master = options['master']
-            else:
-                master = ""
-
-            store = store_factory.buildHBase(zookeepers, master, port, table)
-            reader = reader_factory.buildHBase(store, pysc._jsc.sc())
-            value_reader = value_reader_factory.buildHBase(store)
-            writer = writer_factory.buildHBase(store, table)
-
-        elif backend == 'accumulo':
-
-            # The assumed uri looks like: accumulo://username:password/zoo1, zoo2/instance/table
-            (user, password) = parsed_uri.netloc.split(':')
-            split_parameters = parsed_uri.path.split('/')[1:]
-
-            store = store_factory.buildAccumulo(split_parameters[0],
-                                                split_parameters[1],
-                                                user,
-                                                password,
-                                                split_parameters[2])
-
-            reader = reader_factory.buildAccumulo(split_parameters[1],
-                                                  store,
-                                                  pysc._jsc.sc())
-
-            value_reader = value_reader_factory.buildAccumulo(store)
-
-            writer = writer_factory.buildAccumulo(split_parameters[1],
-                                                  store,
-                                                  split_parameters[2])
-
-        else:
-            raise ValueError("Cannot find Attribute Store for", backend)
-
-        _mapped_cached[new_uri] = _cached(store=store,
-                                          reader=reader,
-                                          value_reader=value_reader,
-                                          writer=writer)
-
-
-def _in_bounds(layer_type, uri, layer_name, zoom_level, col, row):
-    if (layer_name, zoom_level) not in _mapped_bounds:
-        layer_metadata = read_layer_metadata(layer_type, uri, layer_name, zoom_level)
-        bounds = layer_metadata.bounds
-        _mapped_bounds[(layer_name, zoom_level)] = bounds
-    else:
-        bounds = _mapped_bounds[(layer_name, zoom_level)]
-
-    mins = col < bounds.minKey.col or row < bounds.minKey.row
-    maxs = col > bounds.maxKey.col or row > bounds.maxKey.row
-
-    if mins or maxs:
-        return False
-    else:
-        return True
-
-
-def read_layer_metadata(layer_type,
-                        uri,
+def read_layer_metadata(uri,
                         layer_name,
-                        layer_zoom,
-                        options=None,
```

Comment:
They're part of the `uri` structure now: http://geotrellis.readthedocs.io/en/latest/guide/tile-backends.html?highlight=cassandra#backend-uri